### ISSUE TITLE
htmllint can cause error in dump-view

### ISSUE THREAD

**Issue by LOK-Soft** [namics/generator-nitro#122]
if html lint has a certain amount of output dump-view will crash from time to time
```
internal/streams/legacy.js:59
      throw er; // Unhandled stream error in pipe.
      ^

Error: connect ECONNREFUSED 127.0.0.1:55507
    at Object.exports._errnoException (util.js:1018:11)
    at exports._exceptionWithHostPort (util.js:1041:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1086:14)
```

**Comment by ernscht** [comment]
Never noticed untill now ;_)
As far as i understand, the issue is not reproducable, that makes it hard to help.
So I have only questions.

What task do you run?
Whitch nitro version(s), which OS?
How many liniting errors you get when dump-view does not fail?

**Comment by LOK-Soft** [comment]
This issue occured with the newest nitro (generator-nitro 3.1.3) on Mac OS 10.11.6 with node v6.11.1 executing `gulp dump-view`
You have to have about 10 to 20 pages with 5 to 10 errors each, so get the error quite constant.

**Comment by ernscht** [comment]
do you have the opportunity to test 
* with node 8?
* with a different device and on windows?
* give access to your setup somehow?

**Comment by ernscht** [comment]
Seems to be a mac specific issue.

I got access to a sample-project with the issue.
25 runs under windows with node 6 and 25 runs with node 8 - without problems.
Also no problems with 25 runs under ubuntu (windows subsystem for linux)

**Comment by ernscht** [comment]
Using production mode on server should fix this issue. (0260b6c)
Released in version [3.2.0](https://github.com/namics/generator-nitro/releases/tag/3.2.0)
Please reopen if this doesn't fix the issue on your mac.
